submissions/RomanSaiko/html-css-popup/index.html

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
Please add `<meta type="viewport"...` for mobile devices and fix styles
<img width="545" alt="Screenshot 2020-11-03 at 17 32 59" src="https://user-images.githubusercontent.com/42999342/98006363-ce978c80-1dfa-11eb-88d7-6bb20d15a45d.png">

Reply:
Your colleague recommended "better to avoid viewport values that prevent users from resizing documents", so I deleted it